### PR TITLE
Fix alt text stripping accents and single quotes

### PR DESCRIPTION
**Pull request by Tofandel**
The altTextFrom method doesn't work well for non english characters and accents, it also uppercases every word which is weird

### CODE REVIEW

File: src/Helpers/media_library_helpers.php [hunk 1]
```diff
@@ -63,9 +63,6 @@ function bytesToHuman($bytes)
      */
     function replaceAccents($str)
     {
-        if (function_exists('mb_convert_encoding')) {
-            return mb_convert_encoding($str, 'ASCII', 'UTF-8');
```
Comment (Tofandel) [inline src/Helpers/media_library_helpers.php:67]: I had to remove this after adding tests `é` would be converted into `?` instead of `e` which could result in empty filenames

**Review by ifox — APPROVED** [review]
